src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecPlanGraph/Comparison/SkeletonManager.cs

```diff
@@ -22,8 +22,11 @@ public class SkeletonManager
         /// </summary>
         /// <param name="root">Node to construct skeleton of</param>
         /// <returns>SkeletonNode with children representing logical descendants of the input node</returns>
-       public SkeletonNode CreateSkeleton(Node root)
+        public SkeletonNode CreateSkeleton(Node root)
```

Comment:
Remove extra space

Reply:
There was a space missing, so when I formatted the document this extra space was added. It's hard to tell in the diff, but the "p" in public highlighted in red appears one character position too early 
![image](https://user-images.githubusercontent.com/87730006/158485731-53ee40bb-4747-4c01-a796-7f72121e35e6.png)